src/django/api/management/commands/direct_data_load.py

```
@@ -0,0 +1,443 @@
+            action='store_true',
+            help="Skip existing facilities in the database.",
+        )
+        parser.set_defaults(skip_existing=False)
+
+    def handle(self, *args, **options):
+        user_id = options["user_id"]
+
+        try:
+            user = User.objects.get(id=user_id)
+        except User.DoesNotExist:
+            raise ValueError(f"User with ID '{user_id}' does not exist.")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)